docs/content/preview/explore/ysql-language-features/postgresql-compatibility.md

```diff
@@ -16,10 +16,106 @@ YugabyteDB is a [PostgreSQL-compatible](https://www.yugabyte.com/tech/postgres-c
+| Wait-on-Conflict concurrency for predictable P99 latencies | {{<badge/tp>}} | {{<badge/ea>}} | {{<badge/ga>}} | {{<badge/ga>}} |
+| Cost based optimizer. Includes query pushdowns, LSM indexes, and batched nested loop joins for PostgreSQL-like performance. | {{<badge/tp>}} | {{<badge/ea>}} | {{<badge/ga>}} | {{<badge/ga>}} |
+| Use range sharding (ascending) by default | {{<badge/tp>}} | {{<badge/ea>}} | {{<badge/ga>}} | {{<badge/ga>}} |
+| feature |  | {{<badge/tp>}} | {{<badge/ea>}} | {{<badge/ga>}} |
+| feature |  | {{<badge/tp>}} | {{<badge/ea>}} | {{<badge/ga>}} |
+| feature |  |  | {{<badge/tp>}} | {{<badge/ea>}} |
+
+### v2024.1
+
+| EA Feature | Flag |
+| :--- | :--- |
+| Read-Committed isolation | yb_enable_read_committed_isolation=true |
+| Wait-on-Conflict concurrency for predictable P99 latencies | enable_wait_queues=true |
+| Cost based optimizer. Includes query pushdowns, LSM indexes, and batched nested loop joins for PostgreSQL-like performance. | yb_enable_base_scans_cost_model=true<br>yb_bnl_batch_size=1024<br>yb_fetch_row_limit=0<br>yb_fetch_size_limit=1MB |
+| Use range sharding (ascending) by default | yb_use_hash_splitting_by_default=false |
+
+### v2024.2
+
+The following shows the features that are planned for Enhanced Postgres Compatibility Mode for v2024.2 and is subject to change without notice.
+
+| EA Feature | Flag |
+| :--- | :--- | :--- |
+| Feature 1 | flag=false |
+| Feature 2 | flag=true |
+| Feature 3 | flag=false |
+
+The following features are planned to go {{<badge/ga>}} in v2024.2 and will no longer be managed under Enhanced Postgres Compatibility Mode:
+
+- Read-Committed isolation
+- Cost based optimizer
+- Use range sharding (ascending) by default
+
+## Enable enhanced compatibility mode
+
+### YugabyteDB
+
+To enable compatibility mode in YugabyteDB:
+
+- Pass the `enable_pg_parity_early_access` flag to yugabyted when bringing up your cluster.
+
+For example, from your YugabyteDB home directory, run the following command:
+
+```sh
+./bin/yugabyted start --enable_pg_parity_early_access
+```
+
+Note: When enabling the cost models, ensure that packed row for colocated tables is enabled by setting the `--ysql_enable_packed_row_for_colocated_table` flag to true.
+
+### YugabyteDB Anywhere
+
+To enable compatibility mode in YugabyteDB Anywhere v2024.1, see the [Release notes](../../../releases/yba-releases/v2024.1/#highlights).
+
+To enable compatibility mode in YugabyteDB Anywhere v2024.2 or later:
+
+- When creating a universe, turn on the **Enable Enhanced Postgres Compatibility** option.
+
+  You can also change the setting on deployed universes using the **More > Edit Postgres Compatibility** option.
+
+### YugabyteDB Aeon
+
+To enable compatibility mode in YugabyteDB Aeon:
+
+- When creating a cluster, choose the **Early Access** track; the **Enable Enhanced Postgres Compatibility** option is on by default.
```

Comment:
nit: Rather than make the choice a track-specific one, can you more abstractly say any DB version >= 2024.1.0?